packages/components/pie-cookie-banner/src/defs.ts

```diff
@@ -148,7 +181,37 @@ export type DefaultProps = ComponentDefaultProps<CookieBannerProps>;
 export const defaultProps: DefaultProps = {
     hasPrimaryActionsOnly: false,
     defaultPreferences: {},
-    locale: defaultLocale,
+    tenant: Tenant.GREAT_BRITAIN,
+    language: Language.ENGLISH,
     cookieStatementLink: '',
     cookieTechnologiesLink: '',
 };
+
+export const getDefaultLanguageForTenant = (tenant: Tenant): Language => {
```

Comment:
By utlising a `Map` or `Set` above as suggested, this can be completely removed as it just becomes object lookups and perhaps a default when none is found